examples/chef/NEW_CHEF_DEVICES.md

```diff
@@ -0,0 +1,110 @@
+# CHEF devices
+
+Device definitions reside in `examples/chef/devices` as zap and .matter files.
+
+Generally you need to follow these steps to use a new device:
+
+-   Create a zap/matter configuration (forking an existing one is the easiest)
+-   Rename to a final/stable name
+-   Test
+
+## Creating a new zap configuration
+
+The easiest way to get started is using an existing configuration. In this
+example we will use `rootnode_contactsensor_lFAGG1bfRO` as one of the smallest
+devices. Use `./examples/chef/chef.py -h | less` or view
+`./examples/chef/devices/*.zap` for a list of valid devices.
+
+To open the zap GUI for this device, run:
+
+```
+./examples/chef/chef.py -g -d rootnode_contactsensor_lFAGG1bfRO
```

Comment:
```suggestion
./examples/chef/chef.py -gz -d rootnode_contactsensor_lFAGG1bfRO
where 
    `-g` means  run zap GUI display to allow editing of data model
    `-z` means run zap to generate data model & interaction model artifacts
```

Reply:
BTW, `-z` is necessary so that the ZAP callbacks can be generated for next steps building code.

Reply:
I had missed that we use `do_run_zap` for value storage. Checking it now ...

Reply:
This needs some updates. af-gen-event is not needed anymore (will remove it), have to check what else is generated. I was hoping for nothing else needed.

Reply:
I looked at what we generate, I do not think it should be needed: it runs zap generate.py, however that was altered now to just generate a ".matter" file. This is in a separate step in the doc and while it may be convenient here, I believe we probably should just remove this flag completely (maybe in a separate PR just in case something breaks).